app/controllers/concerns/bbb_server.rb

```diff
@@ -88,12 +83,19 @@ def start_session(room, options = {})
       else
         bbb_server.create_meeting(room.name, room.bbb_id, create_options)
       end
-
-      unless meeting[:messageKey] == 'duplicateWarning'
-        room.update_attributes(sessions: room.sessions + 1, last_session: DateTime.strptime(meeting[:createTime].to_s, "%Q"))
+      logger.warn "Duplicated create request" if meeting[:messageKey] == 'duplicateWarning'
```

Comment:
We need the context of the log, room name and user at least if possible

Reply:
Sure,